core/src/main/kotlin/org/evomaster/core/problem/api/service/ApiWsStructureMutator.kt

```diff
@@ -202,14 +202,16 @@ abstract class ApiWsStructureMutator : StructureMutator() {
             }
         }
 
+        individual.individual.addInitializingHostnameResolutionActions(actions = addedInsertions)
+
         // update impact based on added genes
-        if (mutatedGenes != null && config.isEnabledArchiveGeneSelection()) {
-            individual.updateImpactGeneDueToAddedInitializationGenes(
-                mutatedGenes,
-                old,
-                listOf(addedInsertions)
-            )
-        }
+//        if (mutatedGenes != null && config.isEnabledArchiveGeneSelection()) {
```

Comment:
why removing this?

Reply:
This was causing error at impact calculation (`org/evomaster/core/search/EvaluatedIndividual.kt:817`). Also `HostnameResolutionAction` doesn't have any `Gene`s. Is this necessary for impact calculation?

Reply:
@seran no, not necessary. will actually be removed in a different branch. for now, just add comment stating commented out due to not having genes